usecases/organization_usecase.go

```diff
@@ -109,3 +109,23 @@ func (usecase *OrganizationUseCase) DeleteOrganization(ctx context.Context, orga
 	)
 	return nil
 }
+
+func (usecase *OrganizationUseCase) GetOrganizationFeatureAccess(ctx context.Context,
+	organizationId string,
+) (models.OrganizationFeatureAccess, error) {
+	if err := usecase.enforceSecurity.CreateOrganization(); err != nil {
```

Comment:
This one should probably rather be like this
```suggestion
	if err := usecase.enforceSecurity.ReadOrganization(); err != nil {
```
(maybe you need to change the enforceSecurity that's injected, I'm not sure while looking at it on github).
This is because every used in the org should be allowed to read the feature access (since the front will request it)